xicam/plugins/__init__.py

```diff
@@ -377,13 +432,20 @@ def load_element(self, element, candidate_infofile, plugin_info):
                         # break  # If it didn't work once it wont again
                         msg.logError(RuntimeError("An error occurred while loading plugin: %s" % plugin_info.path))
                     else:
-                        plugin_info.categories.append(current_category)
-                        self.category_mapping[current_category].append(plugin_info)
+                        # plugin_info.categories.append(current_category)
+                        # self.category_mapping[current_category].append(plugin_info)
                         self._category_file_mapping[current_category].append(candidate_infofile)
 
                         return True
 
 
+class EntryPointPluginInfo():
```

Comment:
How does this different from `[Entrypoint](https://entrypoints.readthedocs.io/en/latest/api.html#entrypoint-objects)`? What is `categories`?

Reply:
This is a wrapper class to provide a similar interface as the PluginInfo provided by Yapsy. It is only necessary as long as the two discovery mechanisms coexist.

I'll plan to clean this up once we've transitioned plugins to the new system.

The `categories` attribute should reflect the 'plugin type' (i.e. `"CatalogPlugin"`), and is populated by the plugin manager. This is another artifact of the Yapsy design that can be refactored out later.